### PR TITLE
linter updates

### PR DESCRIPTION
**Pull request by dan-wittenberg**
Updates for the latest style guide

### CODE REVIEW

File: manifests/user.pp [hunk 1]
```diff
@@ -54,12 +73,12 @@
   if $home {
     $home_real = $home
   } elsif $name == 'root' {
-    $home_real = $::osfamily ? {
+    $home_real = $facts['osfamily']['family'] ? {
```
Comment (eputnam) [inline manifests/user.pp]: I think `$facts['os']['family']` is what you're looking for here.

File: manifests/user.pp [hunk 2]
```diff
@@ -54,12 +73,12 @@
...
       'Solaris' => '/',
       default   => '/root',
     }
   } else {
-    $home_real = $::osfamily ? {
+    $home_real = $facts['osfamily']['family'] ? {
```
Comment (eputnam) [inline manifests/user.pp]: and here

File: manifests/home_dir.pp [hunk 3]
```diff
@@ -5,6 +5,17 @@
 # [*user*] User that owns all of the files being created.
 # [*sshkeys*] List of ssh keys to be added for this user in this
 # directory
+# 
+# @param user User to update
```
Comment (eputnam) [inline manifests/home_dir.pp]: awesome, love this!

**Comment by tphoney** [comment]
Hi @dan-wittenberg , thanks for the work you have put into this pull request. However due to inactivity and failing unit tests this pull request is being closed. If you want to re-submit this PR with the necessary changes, we will gladly look at the new pull request.

**Pull request closed without merging**